tests/Gedmo/Mapping/SluggableMappingTest.php

```diff
@@ -84,19 +84,19 @@ public function shouldBeAbleToMapSluggableUsingYamlDriver()
 
         $this->assertArrayHasKey('handlers', $config['slugs']['slug']);
         $handlers = $config['slugs']['slug']['handlers'];
-        $this->assertEquals(2, count($handlers));
+        $this->assertEquals(2, is_array($handlers) || $handlers instanceof \Countable ? count($handlers) : 0);
```

Comment:
```suggestion
        $this->assertCount(2, $handlers);
```
The same applies to the other changes with count in tests.